tests/pipelines/test_register_step.py

```diff
@@ -17,7 +21,19 @@
 )  # pylint: enable=unused-import
 
 
-@pytest.mark.usefixtures("clear_custom_metrics_module_cache")
+@pytest.fixture
+def registry_uri_path(tmp_path):
+    previousRegistryUri = mlflow.get_registry_uri()
+    db_tmpdir = tempfile.mkdtemp()
```

Comment:
No need to `mkdtemp()` if we've added the `tmp_path` fixture. We can just reference `tmp_path`.